types/moko/LW001-BG-PRO/gps.schema.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$id": "https://akenza.io/moko/lw001-bg-pro/gps.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "gps",
+  "title": "GPS",
+  "properties": {
+    "latitude": {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/location/schema.json#/$defs/latitude/degrees"
+    },
+    "longitude": {
+      "$ref": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/location/schema.json#/$defs/longitude/degrees"
+    },
+    "pdop": {
+      "title": "Dilution of Precision",
+      "type": "number",
+      "description": "Dilution of Precision"
+    },
+    "timestamp": {
```

Comment:
is there a need to additionally output the timestamp or can it be the one of the sample?